src/test/java/org/folio/rest/utils/ErrorHelperTest.java

```diff
@@ -0,0 +1,45 @@
+package org.folio.rest.utils;
+
+import org.folio.rest.jaxrs.model.Error;
+import org.folio.rest.jaxrs.model.Errors;
+import org.folio.rest.jaxrs.model.Parameter;
+import org.junit.jupiter.api.Test;
+
+import javax.ws.rs.core.Response;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ErrorHelperTest {
+
+  private static final String CONSTRAINT_NAME = "test_constraint";
+
+  @Test
+  void createValidationErrorMessage() {
+    Errors message = ErrorHelper.createValidationErrorMessage("field", "value", "message");
+    assertNotNull(message);
+    assertEquals(1, message.getErrors().size());
+
+    Error error = message.getErrors().get(0);
+    assertEquals("message", error.getMessage());
+
+    assertNotNull(error.getParameters());
+    assertEquals(1, error.getParameters().size());
+
```

Comment:
The test skipped code verification:
`assertEquals("unique code", error.getCode());`